src/UI/SelectList.elm

```diff
@@ -0,0 +1,232 @@
+module UI.SelectList exposing (SelectList, selectList, toEl, withActionBar, withOptions, withSearchField, withSelected)
+
+import Element exposing (Attribute, Element, fill, shrink)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Events as Events
+import Element.Font as Font
+import UI.ActionBar as ActionBar
+import UI.Icon as Icon exposing (Icon)
+import UI.Internal.Palette as Palette
+import UI.RenderConfig exposing (RenderConfig)
+import UI.Text as Text
+import UI.TextField as TextField
+import UI.Utils.ARIA as ARIA
+import UI.Utils.Element as Element
+
+
+type alias SearchOptions object msg =
+    { label : String
+    , searchMsg : String -> msg
+    , currentFilter : Maybe ( String, String -> object -> Bool )
+    }
+
+
+type alias Options object msg =
+    -- FUTURE: sort: Maybe (List object -> List object)
+    { itens : List object
```

Comment:
maybe it's a typo here, would be `items`?

Reply:
It is :portugal: fault!